sites/partners/src/lib/hooks.ts

```diff
@@ -499,9 +499,10 @@ export const createDateStringFromNow = (format = "YYYY-MM-DD_HH:mm:ss"): string
 
 export const useApplicationsExport = (listingId: string, includeDemographics: boolean) => {
   const { applicationsService } = useContext(AuthContext)
+  const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone.replace("/", "-")
```

Comment:
When testing this locally the time on the export is different than the one on the UI. Specifically the export is in Central time (the timezone my browser is in) whereas the UI shows pacific time (I believe that is hard coded in the code)

Reply:
Good catch! My most recent changes should resolve that issue ⌚